spec/features/admin/admin_orders_spec.rb

```diff
@@ -15,37 +15,33 @@
       user = User.create({fullname: "Wolfgang Mozart", display_name: "Wolfie", email: "MCProdigy@gmail.com", password: "password", password_confirmation: "password", phone: "303-675-1234", role: 0})
       beer1 = {name: "Yeti", state: true, description: "big, roasty malt flavor with rich caramel and toffee notes", price: 606, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC")}
       beer2 = {name: "Chocolate Stout", state: true, description: "full-bodied flavor with hints of rich oatmeal and dark chocolate", price: 612, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:54:09 UTC")}
-      @order = Order.create(user_id: user.id, status: "ordered", total: 1218, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:58:15 UTC") )
+      @order = Order.create(user_id: user.id, status: "ordered", total: 1218, created_at: Date.parse("2012-03-25 09:54:09 UTC"), updated_at: Date.parse("2012-03-25 09:58:15 UTC"))
       beer_orders = @order.beers.create(beer1)
       beer_orders2 = @order.beers.create(beer2)
+      @order.order_beers.create(beer_id: beer_orders.id, quantity: 1)
+      @order.order_beers.create(beer_id: beer_orders2.id, quantity: 1)
     end
 
-    xit 'displays the orders' do
-      visit root_path
-      click_link "Login"
-      fill_in "session[email]", with: "group@email.com"
-      fill_in "session[password]", with: "pw"
-      click_button "Login"
+    it 'displays the orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.